ui/package.json

```diff
@@ -16,7 +16,8 @@
   "resolutions": {
     "lodash": ">=4.17.11",
     "braces": ">=2.3.1",
-    "axios": ">=0.19.0"
+    "axios": ">=0.19.0",
+    "eslint": ">=4.18.2"
```

Comment:
We should start a betting pool to find out at what point do we have all the second-level dependencies in resolutions. :cry: